Assets/MixedRealityToolkit/UX/Scripts/Tooltip/ToolTipUtility.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using UnityEngine;
+
+public static class ToolTipUtility {
+
+    /// <summary>
+    /// Used to find a pivot point that is closest to the 
+    /// anchor. This ensures a natural-looking attachment where the connector line
+    /// meets the label.
+    /// </summary>
+
+    const int NumPivotLocations = 8;
+
+    [Serializable]
+    public enum AttachPointTypeEnum
```

Comment:
Same Enum comments as before:
* Separate out to unique cs file for public and make generic
* Remove enum suffix